### ISSUE TITLE
hlist vertically centers far right column, which looks weird

### ISSUE THREAD

**Issue by barbara-sfx** [sphinx-doc/sphinx#5163]
### Problem
- If you use the example of hlist included in the sphinx docs, the hlist vertically centers far right column, which looks weird

#### Procedure to reproduce the problem
```
.. hlist::
   :columns: 3

   * A list of
   * short items
   * that should be
   * displayed
   * horizontally

```

#### Error logs / results

![image](https://user-images.githubusercontent.com/17709550/42653993-f65be000-85cb-11e8-947e-8ceba7856ece.png)

Also our live site: https://docs.signalfx.com/en/latest/reference/analytics-docs/analytics-reference.html

#### Expected results
far right column should just just start at the same level as the first row of other columns

### Environment info
- OS: Mac 10.13.4
- Python version: 2.7.11
- Sphinx version: 1.7.5

**Comment by tk0miya** [comment]
+0: I don't have strong opinion. But adding new style is okay to me.

**Comment by tk0miya** [comment]
Fixed by #5180.
Thank you for reporting!

**Comment by barbara-sfx** [comment]
Thank you for fixing! I look forward to 1.8.
